labscript_utils/__init__.py

```diff
@@ -15,6 +15,7 @@
 import os
 import traceback
 from pathlib import Path
+import importlib
 
 from .versions import get_version, NoVersionInfo
```

Comment:
`NoVersionInfo` unused

Reply:
Can `labscript_utils.version.get_version` and `labscript_utils.versions.VersionException` further down be omitted as well?

Reply:
I'm not seeing my own comments here for some reason. Let me know if you see this!

Other labscript suite code uses those functions. But, I've been going through bits of the rest of the suite removing the ones that are no longer needed. If it turns out that's all of them, then I'll remove it from here. But I suspect there are a few legit cases that might remain.

Reply:
I couldn't find any with a whole-workspace search.

Reply:
I see two uses of `VersionException` in NI DAQmx, none of `get_version`, and actually I was thinking of `check_version()`, there are 77 of those.

Reply:
(overcounting because it includes both the definitions and the uses to be removed by this PR).

So I can delete get_version, but I suspect after going through the remaining version checks I'll probably be removing or very much downsizing the versions module, so I'll get rid of these then based on how that goes

Reply:
With all the other remove-cruft pull requests, these imports (and indeed the whole `versions` module) are now unused by labscript suite code so I've removed them.